### PR TITLE
[Parser] Ignore blanks + send empty cues arr + valid default end #16

### PR DESCRIPTION
**Pull request by goatandsheep**
#16

### CODE REVIEW

File: lib/parser.js [hunk 1]
```diff
@@ -92,7 +93,7 @@ function parseCues (cues) {
 function parseCue (cue, i) {
   let identifier = '';
   let start = 0;
-  let end = 0;
+  let end = 0.01;
```
Comment (osk) [inline lib/parser.js:96]: Why is this?

**Review by osk — CHANGES_REQUESTED** [review]
There's one test that's not running on your branch:
https://github.com/goatandsheep/node-webvtt/blob/master/test/parser.test.js#L258

You can also merge master to see if circleci doesn't pick it up and run the tests automatically.

**Comment by goatandsheep** [comment]
Thank you for pointing that out! I'll look into it

File: lib/parser.js (hunk 1 above)
Comment (goatandsheep) [reply lib/parser.js:96]: End time must be greater than start time.

**Comment by goatandsheep** [comment]
It failed because the caption in the test had no text

**Pull request merged**